topology/Boston University/Boston University ATLAS Tier2/BU_ATLAS_Tier2.yaml

```
@@ -217,4 +217,43 @@ Resources:
           endpoint: atlas-owamp.bu.edu
     VOOwnership:
       ATLAS: 100
+  SLATE_BU:
```

```suggestion
  SQUID_SLATE_BU:
```

@LincolnBryant Maybe something like this to leave room for other SLATE services at BU?

---

LGTM, thanks for that change